src/main/target/NAZE-rev6/AFROMINI.mk

```diff
@@ -0,0 +1 @@
+/home/christian/Development/betaflight/src/main/target/NAZE/AFROMINI.mk
```

Comment:
Still, relative or absolute, should we have target dependencies and references like that? And targets AFROMINI and  BEEBRAIN in both places? That would mean a clash. And are there really a Rev6 variants of those two?